source/client-side-encryption/corpus/corpus-encrypted.json

```diff
@@ -166,7 +166,7 @@
     "algo": "det",
     "method": "explicit",
     "identifier": "altname",
-    "allowed": false,
+    "allowed": true,
```

Comment:
Oddly, these were set to the wrong value in `corpus-encrypted.json` but not `corpus.json`. Since this isn't used for comparison (and the Java driver was using the `allowed` from `corpus.json`) this didn't show up in tests, but showed up when I regenerated.

Reply:
Ack.